packages/venia-ui/lib/components/CartPage/GiftCards/giftCards.css

```diff
@@ -94,12 +79,11 @@
         align-items: start;
         display: grid;
         gap: 1rem;
```

Comment:
These extra styles are redundant, consider removing. I know you didn't touch this line, but the `composes: apply_button` above is also not needed, if you could clean that up as well.

```suggestion
```

Reply:
Good call out.

Reply:
Done in b87b2e757